yb-voyager/cmd/root.go

```diff
@@ -49,7 +49,7 @@ var rootCmd = &cobra.Command{
 Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like setting up source/target, migration workflow etc.`,
 
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		if cmd.CommandPath() == "yb-voyager" || cmd.CommandPath() == "yb-voyager version" {
+		if cmd.CommandPath() == "yb-voyager" || cmd.CommandPath() == "yb-voyager version" || cmd.CommandPath() == "yb-voyager help" {
```

Comment:
Just like the `noLockNeededList` define a list of command paths that do not require export dir validation. And then use the list to return early from the `PersistentPreRun()`.

The list should also include the intermediate commands like `import`, `import data to`, `initiate` etc. Only the leaf commands that actually trigger some action should have the check.